history/app.py

```diff
@@ -16,11 +16,10 @@
 app.add_route('/device/{device_id}/history', DeviceHistory())
 app.add_route('/notifications/history', NotificationHistory())
 app.add_route('/STH/v1/contextEntities/type/{device_type}/id/{device_id}/attributes/{attr}', STHHistory())
-app.add_route('/log',LoggingInterface())
+app.add_route('/log', LoggingInterface())
 
 # Useful for debugging problems in API, it works with pdb
 if __name__ == '__main__':
     from wsgiref import simple_server  # NOQA
-    httpd = simple_server.make_server('127.0.0.1', 8000, app)
+    httpd = simple_server.make_server('127.0.0.1', 8787, app)
```

Comment:
Be careful about change this port. Please, revert to 8000.

Reply:
You are right, I thought that this section was intended for standalone development only, I will return to the current value.

Reply:
I think you all right, but I think It's better not change that.